docs/install.md

```diff
@@ -6,6 +6,10 @@ Flarum is **beta software**. That means it still has some incomplete features an
 Beta is all about fixing these issues and improving Flarum. **Please don't use Flarum in production unless you know what you're doing**. We can’t support you if things go awry. Upgrading to subsequent versions will be possible, but might involve getting your hands dirty. 
 :::
 
+::: tip Quick test drive?
+If you just want to see if Flarum is suitable for you, you can set up a forum in seconds at [Free Flarum](https://www.freeflarum.com) (not affiliated with flarum.org).
```

Comment:
Let's redirect people to the topic first, so that they can review what is currently going on and decide for themselve whether to visit the actual domain. Is that okay?
```suggestion
If you just want to see if Flarum is suitable for you, you can set up a forum in seconds at [Free Flarum](https://discuss.flarum.org/d/7585) (not affiliated with flarum.org).
```

Reply:
If it links to freeflarum.com directly, it would help people to find FF via Google. Is that ok too? :D

Reply:
For now I think we can live with that.